test/WebSites/FiltersWebSite/BasicAuthenticationHandler.cs

```diff
@@ -23,9 +28,9 @@ protected override Task<AuthenticateResult> HandleAuthenticateAsync()
                     new Claim(ClaimTypes.Role, "Administrator"),
                     new Claim(ClaimTypes.NameIdentifier, "John")
                 },
-                Options.AuthenticationScheme));
+                Scheme.Name));
             return Task.FromResult(AuthenticateResult.Success(new AuthenticationTicket(principal, 
-                new AuthenticationProperties(), Options.AuthenticationScheme)));
+                new AuthenticationProperties(), Scheme.Name)));
         }
```

Comment:
Well, its basically a Mock auth scheme for the functional auth filters test, so somewhat meaningful :)